### ISSUE TITLE
Git import does not seem to work but no error is thrown

### ISSUE THREAD

**Issue by mtamboli** [eclipse-che/che#16716]
### Describe the bug

After enabling Code Ready Workspaces, you can create workspace and define the project to be imported.  It shows check marks as if the project is validated but it takes any correct/incorrect URL. When you click create, nothing happens. 

![image](https://user-images.githubusercontent.com/29490950/79988739-fafe4380-847c-11ea-8ada-018700913adf.png)

### Che version

 - [ ] latest
 - [ ] nightly
 - [ ] other: please specify
```
2.1.0.GA :: crw_master/1107 :: che-dev @ 9348 (19) :: che-parent @ 29042 (7.9.2) :: che @ 0ea1a6 (7.9.2) :: codeready-workspaces @ 5661 (2.1.0.GA)
```
### Steps to reproduce

1. Install CRW with Kabanero 0.6.0
2. Enable CRW
3. Create Workspace and select import GitHub project option 
4. It shows projects to be validated (green checkmark) for incorrect URLs
5. Nothing happens after you click create.

### Expected behavior

1. I expect validation to fail for incorrect URL
2. I also expect Create workspace to throw error if there is an issue other than nothing happening.

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [x] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [ ] chectl 
   - [ ] che-operator 
   - [ ] minishift-addon
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] other: please specify

**Comment by sleshchenko** [comment]
Dashboard is not able to clone git projects, it just prepares projects section in devfile and let Theia import projects after workspace start. Don't you see an error from Theia after workspace is started? Like
![Screenshot_20200423_121559](https://user-images.githubusercontent.com/5887312/80081880-3a4f9180-855c-11ea-8653-a1d8aca8906e.png)


So, the provided URL seems to be valid in terms of URL rules but Dashboard handles incorrectly other invalid URLs like `https://as adsad asd@@$#$`
![Screenshot_20200423_121756](https://user-images.githubusercontent.com/5887312/80082068-7edb2d00-855c-11ea-8644-a45f9bafcc15.png)

**Comment by mtamboli** [comment]
@sleshchenko Thanks for looking into it. My observation is 
1. In Che 7.9.2 - There was no error when URL was selected but you could not create the workspace.
2. In Che 7.9.3 - It lets you move forward with any URL (correct or incorrect) and you see error after the workspace is created.

So part of the problem I reported is already fixed  in 7.9.3. But I still think that it should be able to check the URL for validity prior to creating workspace even if it is not cloning. Especially the green tick marks makes you think that this is correct repo. So in this case, I would like to know prior that this is not going to work.

![image](https://user-images.githubusercontent.com/29490950/80112364-06ba3a80-854f-11ea-86e5-c3a52b3a004a.png)
